### ISSUE TITLE
Strength/weaknesses are wrong

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pokedex.org#23]
This may require a database fix. Reported on Twitter:

> Hi, I have got some feedback for your excellent work on https://t.co/yLcd9ETTDo. Unfortunately you did not consider that by default Ground moves do not hit Flying type or Fighting does not hit Ghost. Neither did you respect that Eelektross has no weaknesses because of Levitate.

**Comment by Anguspokemon** [comment]
That is true  because as a ghost type your see through and fighting moves would just go through it. And since flying types would be up in the air, ground type moves wouldn't affect it, as in earthquake wouldn't hit it.
